nat-lab/tests/test_mesh_plus_vpn.py

```diff
@@ -365,6 +365,21 @@ async def test_vpn_plus_mesh(
         async with Ping(connection_alpha, beta.ip_addresses[0]).run() as ping:
             await testing.wait_long(ping.wait_for_next_ping())
 
+        # Testing if the VPN node is not cleared after disabling meshnet. See LLT-4266 for more details.
```

Comment:
Question: does this also need to validate that other connection states are not fired? I can't tell, so just checking

Reply:
No, this test is specifically needed just in case something's wrong with the code and it clears the VPN node when it shouldn't